appcenter-data/Data.d.ts

```diff
@@ -47,18 +47,22 @@ export interface RemoteOperationListener {
     onRemoteOperationCompleted?: (remoteOperationCompletedData: RemoteOperationCompletedData) => void;
 }
 
+export interface DataSyncError {
+    message: string;
+}
+
 export interface RemoteOperationCompletedData {
-    id: string;
-    eTag: string;
-    partition: string;
+    id?: string;
+    eTag?: string;
+    partition?: string;
     operation: string;
-    error: Map<string, string> | null;
+    error?: DataSyncError;
 }
- 
+
 export function isEnabled(): Promise<boolean>;
 export function setEnabled(enabled: boolean): Promise<void>;
 export function setRemoteOperationListener(listener: RemoteOperationListener): Promise<void>;
-export function setTokenExchangeUrl(tokenExchangeUrl: string): void; 
+export function setTokenExchangeUrl(tokenExchangeUrl: string): void;
```

Comment:
We don't expose that since it cannot work on Javascript (too late).
Needs to be called in native code.

Reply:
We need to remove this from JS and bridge too, I think I told Jacky about it but this may have been lost.

Reply:
Remove this API from d.ts

Reply:
changed :)